app/lib/appointment_importer.rb

```diff
@@ -0,0 +1,41 @@
+class AppointmentImporter
+  FAKE_DATE_OF_BIRTH = '1950-01-01'.freeze
```

Comment:
Yeah I did wonder about this but decided on this approach or sheer consistency of the imported data. The values for that column have a pretty low cardinality so the quality of stuff coming out from there is kinda low unfortunately.